.github/workflows/unit-tests.yml

```diff
@@ -59,7 +59,6 @@ jobs:
             extra_loc: "zh_CN"
           - name: "Future infer strings"
             env_file: actions-311.yaml
-            pattern: "not slow and not network and not single_cpu"
```

Comment:
_Would_ be nice to keep `not single_cpu` since by default the tests will run with `-n auto`

Reply:
Hmm, I might be misunderstanding how this works in our setup. The main CI builds are also not using any pattern, right? (for example `Unit Tests / ubuntu-latest actions-310.yaml`) 
And I thought that in that case of no pattern, the scripts automatically set up one test run in parallel without the single_cpu tests, and one test run single threaded with the single_cpu ones?

Reply:
Ah sorry I was incorrect; you have it right! (We never really had a custom job with no patter so I forgot that's how this was set up)